### ISSUE TITLE
Refactor: clarify namespace when referencing drawing types

### ISSUE THREAD

**Issue by swharden** [ScottPlot/ScottPlot#1960]
Some types (e.g., `Color`) are present in multiple namespaces (System.Drawing, ScottPlot, Maui.Graphics, and SkiaSharp) and referencing them in code just as `Color` results in ambiguity that makes it difficult to refactor toward ScottPlot 5 (#1955).

This issue will track refactoring ScottPlot4 to avoid `using System.Drawing` wherever possible.

I won't modify the cookbook.

**Comment by swharden** [comment]
A better solution is to leave things the way they are and exclude `ScottPlot.Color` and dependent classes from ScottPlot 4 using conditional references and custom flags

* https://markheath.net/post/csproj-conditional-references
* https://stackoverflow.com/questions/4992511/adding-custom-information-to-csproj-files
* https://stackoverflow.com/questions/43210602/custom-property-on-projectfile-preset-by-solution
* https://stackoverflow.com/questions/45622882/can-i-pass-compilation-constants-to-a-project-reference

### ScottPlot.Common csproj

ScottPlot.Color is conditional

```xml
<ItemGroup Condition="'$(RemoveColor)' == 'true'">
    <Compile Remove="Color.cs" Condition="" />
</ItemGroup>
```

### ScottPlot4 csproj

ScottPlot.Color is not included in the project

```xml
<ItemGroup>
    <ProjectReference Include="..\..\Common\ScottPlot.Common\ScottPlot.Common.csproj">
        <Properties>RemoveColor=true</Properties>
    </ProjectReference>
</ItemGroup>
```

### ScottPlot5 csproj

ScottPlot.Color is available

```xml
<ItemGroup>
    <ProjectReference Include="..\..\Common\ScottPlot.Common\ScottPlot.Common.csproj" />
</ItemGroup>
```

NUnit tests are complaining about this... but it's a start

**Comment by swharden** [comment]
Actually we don't need a csproj at all for the common library.

ScottPlot4.csproj can just:
```xml
<ItemGroup>
    <Compile Include="..\..\Common\ScottPlot.Common\**\.cs" />
    <Compile Remove="..\..\Common\ScottPlot.Common\Color.cs" />
</ItemGroup>
```

**Comment by swharden** [comment]
resolved in #1963
